docs/search relevance/README.md

```diff
@@ -24,34 +24,43 @@ There's a lot of interesting discussion to be had about different search modalit
 
 We're collecting data in two distinct ways:
 
-- **Explicit data collection:** users are told that they are part of an experiment, and providing us with information is their primary motive.
 - **Implicit data collection:** users know that their data might be used to understand aggregate user behaviour, but their behaviour on the site is driven by another motive.
 
-Some of the data we use to measure search relevance is collected in sessions with internal users who know the collection well and the kind of things that users might be looking for, following a set of instructions to obtain data in a useful format.
+The larger of the two major datasets we're using is passively collected while users carry out searches and look through pages of results. We track the query string, the page number, the works they click on, and whether they're on the Wellcome staff network (we want to be able to distinguish between internal and external users). This data is aggregated under an anonymous session id.
 
-This data is small (20-50 rows per person, per session, with ~10 people taking part), but can be deeply contextualised by talking to the people who took part.
+We can collect lots of this data (100,000s of actions logged so far), but it lacks any contextual information and unpicking a user's motivation for an action is hard.
 
-The other major dataset is passively collected while users carry out searches and look through pages of results. We track the query string, the results page number, the works they click on, and whether they're on the Wellcome staff network (we want to be able to distinguish between internal and external users). This data is aggregated under an anonymous session id.
+- **Explicit data collection:** users are told that they are part of an experiment, and providing us with information is their primary motive.
 
-We can collect lots of this data (100,000s of actions logged so far), but it lacks any contextual information and unpicking a user's motivation for an action is hard.
+Some of the data we've used to measure search relevance is collected in sessions with internal users who know the collection well and the kind of things that users might be looking for, following a set of instructions to obtain data in a useful format.
```

Comment:
```suggestion
Some of the data we've used to measure search relevance is collected in sessions with internal users who know the collection well and the kind of things that people / researchers might be looking for, following a set of instructions to obtain data in a useful format.
```

The language of a user feels either dehumanising or too abstract... hopefully we can come up with a language around specific use cases, but I still feel person is a good abstraction of someone interested in our wares.